.gitignore

```diff
@@ -1,5 +1,5 @@
 /vendor
 /composer.lock
-/bin/coveralls
-/bin/phpunit
-/bin/test-reporter
+/bin/
+.idea
+.vagrant
```

Comment:
* `.idea`
* `.vagrant`

Should probably go in your local global git ignore rather than the project 
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore